src/reducers/system/mediaPicker/collectionQueryResults.js

```diff
@@ -1,36 +1,33 @@
 import { FETCH_MEDIAPICKER_COLLECTION_SEARCH, MEDIA_PICKER_TOGGLE_MEDIA_IN_LIST, RESET_MEDIAPICKER_COLLECTION_SEARCH } from '../../../actions/systemActions';
-import { createAsyncReducer } from '../../../lib/reduxHelpers';
+import { createAsyncReducer, concatPrevAndNext } from '../../../lib/reduxHelpers';
 
 const initialState = {
-  args: { type: 0, mediaKeyword: null },
+  args: { type: 1, mediaKeyword: null },
```

Comment:
What does `1` mean in this context?

Reply:
I'll make a constant, but essentially this preserves/determines the tabular setting of collections from sources query settings in the MediaPicker.